load.php

```diff
@@ -169,3 +171,27 @@ function load_plugins() {
 		ElasticPress_Integration\bootstrap();
 	}
 }
+
+/**
+ * Get a globally configured instance of the AWS SDK.
+ */
+function get_aws_sdk() {
+	static $sdk;
+	if ( $sdk ) {
+		return $sdk;
+	}
+
+	$params = [
+		'region'   => HM_ENV_REGION,
+		'version'  => 'latest',
+	];
+
+	if ( defined( 'AWS_KEY' ) ) {
+		$params['credentials'] = [
+			'key'    => AWS_KEY,
+			'secret' => AWS_SECRET,
+		];
+	}
+	$sdk = new \Aws\Sdk( $params );
```

Comment:
I haven't seen `static` used in a normal function before. To clarify my understanding from the docs, declaring a variable `static` _inside_ the function means the value of the variable will be preserved, and when the function is run again, used? So in this case, it lets the function cache the value of `$sdk`?

Reply:
Exactly, it's just an easy way to do a one-time-set cache